INSTALL.md

```diff
@@ -0,0 +1,10 @@
+# Install Zync
+
+Download this repository and run `bundle install`.
+
+Install PostgreSQL with `brew install postgresql`
+Start its service with `brew services start postgresql`
+
+Make sure that it is running. See the services running with `brew services list` and `postgresql` should be in that list with status `started` and looking green. If you see an orange `orange` started, that means that it is trying to start but it is having errors, which you can see with more detail in the logs with `tail -f /usr/local/var/log/postgres.log`.
```

Comment:
I <3 the tip about the orange started!! great that you're including it here - I can see that I could have been banging my head against the wall for a couple of hours having been misguided by that "started" message.

Reply:
That is exactly what happened to me 😂